### PR TITLE
Experiment/prte jsrun

### PR DESCRIPTION
**Pull request by andre-merzky**
This branch contains numerous fixes for prrte and summit, which increase stability and scale, and also improve profiling and logging.  Before merging, several things need cleaning up:

  - reconsider the default sub-agent configuration for prrte
  - sync agent configs for jsrun and prrte
  - reset cache and timeout values to defaults
  - reduce dvm verbosity (`-debug`)

### CODE REVIEW

File: src/radical/pilot/configs/resource_ornl.json [hunk 1]
```diff
@@ -316,10 +310,10 @@
         "lrms"                        : "LSF_SUMMIT",
         "lfs_per_node"                : "/tmp",
         "agent_type"                  : "multicore",
-        "agent_config"                : "default",
-        "agent_scheduler"             : "CONTINUOUS",
+        "agent_config"                : "summit_sa",
```
Comment (mturilli) [inline src/radical/pilot/configs/resource_ornl.json]: Same here?

File: src/radical/pilot/configs/resource_ornl.json [hunk 2]
```diff
@@ -316,10 +310,10 @@
...
+        "agent_scheduler"             : "HOMBRE",
```
Comment (mturilli) [inline src/radical/pilot/configs/resource_ornl.json]: Not sure we want this scheduler as the default one.

File: src/radical/pilot/agent/lm/prte.py [hunk 3]
```diff
@@ -276,14 +289,14 @@ def construct_command(self, cu, launch_script_hop):
             map_flag += ' -host %s' % hosts.rstrip(',')
 
         # Additional (debug) arguments to prun
-        debug_string = ''
+        debug_string = '-verbose'
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/pilot/agent/lm/prte.py]: Thanks, fixed.

File: src/radical/pilot/agent/staging_output/default.py [hunk 4]
```diff
@@ -153,6 +153,21 @@ def _handle_unit_stdio(self, unit):
...
+            # [batch3:122527] JOB [3673,4] EXECUTING
+            with open(unit['stderr_file'], 'r') as stderr_f:
+
+                for line in stderr_f.readlines():
+                    line = line.strip()
+                    if not line:
+                        continue
+                    if line[0] == '[' and line.endswith('EXECUTING'):
+                        elems = line.replace('[', '').replace(']', '').split()
+                        tid   = elems[2]
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/pilot/agent/staging_output/default.py]: If you don't mind, lets leave this for now as that code is supposed to move (and change) anyway, as we discussed.  I'll open a ticket to keep track of the comment.

File: src/radical/pilot/configs/agent_summit_sa.json [hunk 5]
```diff
@@ -1,51 +1,90 @@
 
+# this is the default agent config which is used if no other config is
+# specified for an agent.  It contains the minimal set of settings required for
+# a functional pilot agent, without any component redundency.
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/pilot/configs/agent_summit_sa.json]: Right - I'll clean the configs.  Those settings, and changes in the examples and such, are not supposed to be merged anyway, but are really just for the experiment.  The problem is that I did not keep development out of the experiment branch... :-/

File: src/radical/pilot/configs/resource_local.json [hunk 6]
```diff
@@ -28,7 +28,7 @@
         "rp_version"                  : "local",
         "virtenv_mode"                : "create",
         "python_dist"                 : "default",
-        "cores_per_node"              : 8,
+        "cores_per_node"              : 12,
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_local.json]: RP is able to fake cores and GPUs locally, so this is actually not a problem:
https://github.com/radical-cybertools/radical.pilot/blob/devel/src/radical/pilot/agent/rm/fork.py#L34

:-D

File: src/radical/pilot/configs/resource_ornl.json [hunk 7]
```diff
@@ -257,7 +257,7 @@
         "lrms"                        : "LSF_SUMMIT",
         "lfs_per_node"                : "/tmp",
         "agent_type"                  : "multicore",
-        "agent_config"                : "default",
+        "agent_config"                : "summit_sa",
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_ornl.json]: This is actually back to `default` now.

File: src/radical/pilot/configs/resource_ornl.json (hunk 1 above)
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_ornl.json]: same here...

File: src/radical/pilot/configs/resource_ornl.json (hunk 2 above)
Comment (andre-merzky) [reply src/radical/pilot/configs/resource_ornl.json]: ah, thanks - fixed.

**Pull request merged**